cfgov/jinja2/v1/_includes/templates/nav/primary-nav.html

```diff
@@ -45,15 +37,7 @@
         {%- endfor %}
     </ul><!-- END .primary-nav_top-level-list -->
 
-    {% do global_dict.update(
-        {
-            'global_eyebrow': {
-                'is_in_list': true
-            }
-
-        })
-    %}
-    {% set value = global_dict.global_eyebrow %}
-    {% include 'molecules/global-eyebrow.html' with context %}
+    {% from 'molecules/global-eyebrow.html' import render as eyebrow with context %}
+    {{ eyebrow() }}
```

Comment:
I was following @kurtw's example in https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/browse-filterable/index.html#L29. I never know which is the right way. Kurt, can you fill us in?